File: Assets/Network/Scripts/IPAddress.js
#pragma strict
import UnityEngine.UI;
import UnityEngine.Networking;

private var ipText : UI.Text;
private var ipAddress : String;

function Start () {
	ipText = GetComponent(UI.Text);
}

function Update () {
	ipAddress = Network.player.ipAddress;
	ipText.text = "IP Address: " + ipAddress;
}

File: gear-vr-leap-mobile/Assets/gear-vr-leap/Scripts/IpAddress.cs
using UnityEngine;
using UnityEngine.UI;
using System.Collections;

public class IpAddress : MonoBehaviour {
	Text ipText;
	// Use this for initialization
	void Start () {
		ipText = GetComponent<Text>();
	}
	
	// Update is called once per frame
	void Update () {
		ipText.text = "IP Address: " + Network.player.ipAddress;
	}
}
